app/controllers/password_resets_controller.rb

```diff
@@ -0,0 +1,23 @@
+class PasswordResetsController < ApplicationController
+  def show
+    user = User.where(token: params[:id]).first
```

Comment:
Again, we want to use `find_by` rather than `where().first` - make sure you know the difference.